### PR TITLE
Support tunable=False for HyperModels

### PR DESCRIPTION
**Pull request by omalleyt12**
When `tunable=False`, default values or values that were registered in advance will be used

### CODE REVIEW

File: kerastuner/engine/hypermodel.py [hunk 1]
```diff
@@ -19,13 +19,41 @@
 
 
 class HyperModel(object):
+    """Defines a searchable space of Models and builds Models from this space.
+
+    Attributes:
```
Comment (fchollet) [inline kerastuner/engine/hypermodel.py]: `# Attributes`

File: kerastuner/engine/hypermodel.py [hunk 2]
```diff
@@ -19,13 +19,41 @@
...
+
+        # Arguments:
+            hp: A `HyperParameters` instance.
+
+        # Returns:
+            A model instance.
+        """
         raise NotImplementedError
 
+    def _tunable_aware_build(self, hp):
```
Comment (fchollet) [inline kerastuner/engine/hypermodel.py]: We may want to use a more generic name, such as `_build_wrapper` or `_build_impl`, as we may need to add more functionality to it later.

File: kerastuner/engine/hypermodel.py (hunk 1 above)
Comment (omalleyt12) [reply kerastuner/engine/hypermodel.py]: Done

File: kerastuner/engine/hypermodel.py (hunk 2 above)
Comment (omalleyt12) [reply kerastuner/engine/hypermodel.py]: Done

**Comment by omalleyt12** [comment]
Thanks for the review!

**Pull request merged**